x-pack/test/fleet_api_integration/apis/epm/update_assets.ts

```diff
@@ -342,6 +342,10 @@ export default function (providerContext: FtrProviderContext) {
             id: 'sample_ml_module',
             type: 'ml-module',
           },
+          {
+            id: 'sample_tag',
```

Comment:
The ID should be `b7b33280-26a0-11ec-99c4-6b459b189377`

Reply:
I'll update sample_tag.json, sample_tag will be the real id, no need for id inside attributes